### ISSUE TITLE
License GPLv3 is not appropriate

### ISSUE THREAD

**Issue by hfiguiere** [500px/500pxPublisher.lrplugin#1]
TL;DR use LGPLv3 instead of GPLv3

The long version.

Disclaimer: I am Not A Lawyer. But I do have a good understanding of FLOSS, licenses and GPL.

The plugin is basically "linking" GPLv3 code to non-GPL compatible code (Lightroom). Therefor, implicitely, any third party shipping the 500pxPublisher plug-in would be in violation of the GPLv3, which would be copyright infringement.

While it could be argued that this is not the case, it is not nice.

The LGPLv3 was actually designed with that use case in mind and would preserve the copyleft nature of the license (the requirement to provide the source code) while allowing to "link" the code to Lightroom.

Other options would include using a non copyleft license like Apache v2, MIT or BSD.
